### PR TITLE
Add FAQ page

### PR DESCRIPTION
**Pull request by falbru**
image

Still some console errors and warnings I have to fix, but other than that it's good

### CODE REVIEW

**Comment by itsisak** [comment]
I managed to remove some of the errors I got in console log with remove the graphQL `Question_Query` since you already the same fields in `faq.tsx` I then just moved the tsx from `Question.tsx` straight into where it was used in `faq.tsx` although that might not be necessary (but I think its nice anyways since it would otherwise be a very small component)

File: pages/faq.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import FlexItem from '../components/Styled/FlexItem';
+
+const Title = styled('h1')`
+  font-weight: bold;
+  font-smoothing: antialiased;
+  font-size: 3rem;
+  margin-bottom: 1rem;
+`;
+
+const faq = ({
```
Comment (itsisak) [inline pages/faq.tsx:21]: Component should be capitalized?

**Comment by falbru** [comment]
Great comments! I have changed the code according to your feedback!

**Comment by falbru** [comment]
Looks like the `aria-controls` warning is caused by the Collapse-component in nextui-org. We should probably upgrade to the newest versions of next and nextui-org soon...

**Comment by itsisak** [comment]
I see, I looked into upgrading nextui-org to v2, but that requires configuring tailwind, which felt unnecessary, but maybe we should then..

**Review by itsisak — APPROVED** [review]
Looks good!

**Pull request merged**